school_42/d05/ex08_ft_strupcase.py

```diff
@@ -0,0 +1,25 @@
+# Create a function that transforms every letter of every word to uppercase.
+
+def create_uppercase(data_str):
+    upper_str = ""
+    for i in data_str:
+        if ord(i) >= 97 and ord(i) <= 122:
```

Comment:
please, make it more clear. I know nothing about 97 and 122, i guess that you can replace it by  ord('a')/ord('z') and etc. Or add another variable which will be looked "a_char = ord('a')".